tests/integration/panel/empty-page-tests.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
```

Comment:
is there really a need for this to be its own file? cant we just add an empty page to the main file